### DISCUSSION TITLE
Retain font size between launches

### DISCUSSION THREAD

**Discussion by mikeerickson** [spatie/laravel-ray#48] [Ideas]
It is great that I can zoom out, but could this be configured such that it will maintain the zoom level between launches? 

Or, perhaps a font size setting?

**Comment by freekmurze** [comment]
In my testing it seems that the zoom level is retained between launches on macOS. On which platform are you?

**Reply by mikeerickson** [reply]
@freekmurze Hmm, I have tried to reproduce and it is indeed retaining zoom level.
